scss/styles.scss

```diff
@@ -1,3 +1,7 @@
-.title {
-  color: red;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+%roboto {
+	font-family: 'Roboto', sans-serif;	
```

Comment:
Since the same font family applies to the whole app, set it in the equalizer under the `*`